### PR TITLE
synfigstudio: 1.0.1 -> 1.0.2

### PR DESCRIPTION
**Pull request by leenaars**

### CODE REVIEW

**Comment by leenaars** [comment]
Interestingly, it built locally without a problem, but when pushed to CI it failed. Maybe some weird consequence of using nix-prefetch-url. I'm curious what happens now.

**Comment by cillianderoiste** [comment]
Hi @leenaars, thanks for working on this! Perhaps using fetchFromGithub with the releases from github will work better e.g. https://github.com/synfig/synfig/releases

**Comment by leenaars** [comment]
That works, thanks. Should be ready to merge.

**Comment by domenkozar** [comment]
cc @cillianderoiste

File: pkgs/applications/graphics/synfigstudio/default.nix [hunk 1]
```diff
@@ -1,28 +1,46 @@
...
+       owner  = "synfig";
+       rev    = version;
+       sha256 = "09ldkvzczqvb1yvlibd62y56dkyprxlr0w3rk38rcs7jnrhj2cqc";
     };
+
+    postUnpack = "sourceRoot=\${sourceRoot}/ETL/";
+
+    preConfigure = "autoreconf --install --force";
+    
+    buildInputs = [ autoconf automake ];
```
Comment (globin) [inline pkgs/applications/graphics/synfigstudio/default.nix]: Please use autoreconfHook and remove the `preConfigure` and the other buildInputs here.

File: pkgs/applications/graphics/synfigstudio/default.nix [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
     };
 
+    postUnpack = "sourceRoot=\${sourceRoot}/synfig-core/";
+
+    preConfigure = ''
+      libtoolize --copy
+      chmod +w libltdl/configure
+      autoreconf --install --force
+    '';
+
```
Comment (globin) [inline pkgs/applications/graphics/synfigstudio/default.nix]: See above

**Comment by cillianderoiste** [comment]
Hi @leenaars, could you take a look at @globin's feedback. Thanks!

**Comment by leenaars** [comment]
@globin: I made the changes as suggested, and for me it compiles locally without any problem. But for some reason or another the dependencies are built, and thus the continuous integration runs out of time. Is there anything I can do?

**Comment by cillianderoiste** [comment]
Thanks @leenars, it looks like you missed one of the comments. I can build locally to test, when it's ready. Cheers!

**Comment by vcunat** [comment]
Let me end the ping-pong. Cleaned, tested it starts and pushed.

**Pull request closed without merging**